docs/conf.py

```diff
@@ -13,6 +13,7 @@
 import os
 import sys
```

Comment:
1. Are we actually running those python code? How do we ensure that they are runnable and will not be breaking?
2. Would you mind include the link of the render files for the doc? I am curious about how we transform the .py into tutorial

Reply:
1. They are being run as part of internal CI. 
2. This is the GH pages PR: https://github.com/apple/coremltools/pull/1883; Link to the docs being rendered: https://u-simha.github.io/coremltools

Reply:
specifically here: https://u-simha.github.io/coremltools/source/coremltools.optimize.torch.examples.html

Reply:
Awesome! Now it makes much more sense reading those python files :)